packages/wallet/src/components/Menu.tsx

```diff
@@ -0,0 +1,128 @@
+import { Link } from '@/ui/Link'
+import { Logo } from '@/ui/Logo'
+import { Disclosure, Transition } from '@headlessui/react'
+import { cx } from 'class-variance-authority'
+import { useRouter } from 'next/router'
+import { Fragment } from 'react'
+import { Bars } from './Icons/Bars'
+import { Home } from './Icons/Home'
+import { X } from './Icons/X'
+
+type NavigationItemProps = {
+  name: string
+  href: string
+  icon: JSX.Element
+}
+
+const navigationItems: NavigationItemProps[] = [
+  {
+    name: 'Home',
+    href: '/',
+    icon: <Home className="h-6 w-6" />
+  },
+  {
+    name: 'Accounts',
+    href: '/accounts',
+    icon: <Home className="h-6 w-6" />
+  },
+  {
+    name: 'Developer Account',
+    href: '/developer',
+    icon: <Home className="h-6 w-6" />
+  },
+  {
+    name: 'Grants',
+    href: '/grants',
+    icon: <Home className="h-6 w-6" />
+  },
+  {
+    name: 'Settings',
+    href: '/settings',
+    icon: <Home className="h-6 w-6" />
+  }
+]
+
+export const Menu = () => {
+  const { pathname } = useRouter()
```

Comment:
pathname should be pathName

Reply:
This is what `useRouter` is returning.

https://nextjs.org/docs/api-reference/next/router#router-object